api/src/Entity/OAuthState.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\OAuthStateRepository;
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Database table used for storing valid OAuth state tokens and their expiration dates.
+ * Definitely not exposed on the API, this is only for internal security checking.
+ */
+#[ORM\Entity(repositoryClass: OAuthStateRepository::class)]
+class OAuthState extends BaseEntity {
```

Comment:
An index on both `$state` and `$expireTime` would be nice

Reply:
Thanks, fixed.